examples/sinatra/webvalve/fake_twitter.rb

```diff
@@ -1,6 +1,4 @@
 class FakeTwitter < WebValve::FakeService
-  URL = 'http://faketwitter.test'.freeze
```

Comment:
we can't do this sorta thing anymore. we'd have to put this URL into lib or some other non autoloaded path. i was never a huge fan of this approach, so i'm not worried about it.